src/OmniSharp.Abstractions/Eventing/IEventEmitterExtensions.cs

```diff
@@ -68,7 +74,8 @@ public static void ProjectInformation(this IEventEmitter emitter,
                 References = references.Select(hashed => hashed.Value),
                 FileExtensions = fileExtensions.Select(hashed => hashed.Value),
                 FileCounts = fileCounts,
-                SdkStyleProject = sdkStyleProject
+                SdkStyleProject = sdkStyleProject,
+                ProjectFilePath = projectFilePath
```

Comment:
The ProjectConfigurationMessage is used for some rudimentary telemetry. Passing ProjectFilePath may include some PII. You would likely want to add a new Message type for ProjectLoadFinished.